### DISCUSSION TITLE
sync trakt liked list to plex watchlist and keeping plex watchlist in sync with trakt liked list?

### DISCUSSION THREAD

**Discussion by nmbrg** [Taxel/PlexTraktSync#1728] [Q&A]
I have a couple of questions i hope will get answered here before i try out PlexTraktSync. :)

If i sync "liked lists" from trakt to plex, will it add the liked list in trakt to plex watchlist or will it get added in some some other list or place in plex?

If the list is a modified "trakt anticipated movies" list (or anticipated tv series), created on mdblist, will it keep the watchlist in plex updated with that trakt list? and by that i mean, if something is removed from the trakt list, will it remove it from the plex watchlist? or will it only add items and never remove them?

And if something gets added to the plex watchlist from the liked list on trakt, that isnt possible to remove from the trakt liked list, that i do not want on the plex watchlist, is it possible to remove that movie/series without making it reappear after next sync on the plex watchlist?

**Comment by simonc56** [comment]
Liked lists and watchlist are separated things. They are never mixed in Plex.
The script creates new lists in Plex based on your trakt liked lists.

**Reply by nmbrg** [reply]
Ok, so if i want to populate the watch list in plex i have to use the watch list in trakt?

And about the other questions?

**Reply by simonc56** [reply]
Items can be added and removed in lists.
You can try the script with `--dry-run` argument to see what happen without effective changes.

**Reply by nmbrg** [reply]
I was hoping to not need to waste several hours to create trakt list, install a docker version with plextraktsync creating docker compose and config.yml and then needing to revert everything (removing added stuff in plex watchlist, overseerr, radarr/sonarr and removing container etc) just to answer those simple question that someone with the knowledge could easily answer in a couple of seconds. 

But since it can’t sync a liked list to plex watchlist I’m no longer interested. Because that was what I wanted (needed) it to do. Since I wanted to create a list at mdblist, sync that to trakt. Then get that list synced to plex watchlist. Then overseerr would add the items from plex list to radarr/sonarr. 


so I guess I’m now on a mission to find other alternatives.

the key thing here is that I want notifications from items added thru overseerr from lists as I get from those manually added.

anyway, thanks for your time.

**Reply by simonc56** [reply]
I talked about *dry run* option, so you don't have to revert everything.

I don't understand why you don't use trakt watchlist to sync to plex watchlist. That's the only way to update Plex watchlist with this script.

Also, radarr can read any trakt or mdb list, so you could rethink your workflow.

**Reply by nmbrg** [reply]
Yes, i think i was a bit unclear. I wanted to test the whole flow, and that is not doable with dry-running plextraktsync since that won’t show if the rest of the syncs/imports etc works. 

Because I do not want to manually populate a list (trakt watchlist can only be populated manually afaik), and hence I’m using mdblist to make a list that has certain criteria (anticipated, year, genre, language, channel/service etc).

using radarr/sonarr directly for importing list would only make me get notification on everything, even those items I haven’t requested myself in overseerr.

**Comment by ericvlog** [comment]
if only sync to trakt likes lists, no sync likes list "item" to watchlist ......then i no see any benefit doing so?

I saw it create a playlist in my plex same as likes lists....but it need library have the movies inside the likes lists.

**Reply by CpHeat** [reply]
> if only sync to trakt likes lists, no sync likes list "item" to watchlist ......then i no see any benefit doing so?
> 
> I saw it create a playlist in my plex same as likes lists....but it need library have the movies inside the likes lists.

Same here, I use CouchMoney for having automated recommendations, and it creates Trakt lists (but can't populate my Trakt watchlist)
Now if I import these lists in Plex, they will just import what I already have, so I don't care for it at all
Still the sync for rating from Plex to Trakt is an excellent feature but it is a shame it can't import Trakt lists as Plex watchlist, that would be AWESOME
